test/correctness/simd_op_check.h

```diff
@@ -281,6 +282,10 @@ class SimdOpCheckTest {
                 }
 
                 error_file.close();
+
+                // Occasionally useful for debugging: stop at the first failure.
+                // std::cerr << error_msg.str();
+                // exit(-1);
```

Comment:
Intended to submit this?

BTW, this is what I use HL_SIMD_OP_CHECK_FILTER for.

Reply:
Yeah, I did intend to -- the issue here is that even with HL_SIMD_OP_CHECK_FILTER you can run several tests (different vectorize widths, different factors, etc.). I'll just remove these if it's a nuisance, but I thought it was a handy reminder of where to insert these checks.